plugins/teams/commands/health-check-regressions.md

```
@@ -170,48 +172,49 @@ For each component (from `components.*.summary`):
-- **Data Scope**: Notes which regressions are included based on date filtering
+- **Data Scope**: Notes which regressions are included based on view and date filtering
+  - Regressions are filtered to the specified view (open/closed status comes from the view)
   - For GA'd releases: Regressions within development window (start to GA)
   - For in-development releases: Regressions from development start onwards
 
 ## Examples
 
-1. **Summarize all regressions for a release**:
+1. **Summarize all regressions for a view**:
 
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)